### DISCUSSION TITLE
Package secret_prov_gramine was not found in the pkg-config search path.

### DISCUSSION THREAD

**Discussion by MatheusB4** [gramineproject/gramine#1528] [Q&A] [answered]
Hi,

I was following the tutorial on https://gramine.readthedocs.io/en/stable/tutorials/pytorch/index.html and came
across the following error when using `make SGX=1` on the ~/gramine/CI-Examples/ra-tls-secret-prov directory:

Package secret_prov_gramine was not found in the pkg-config search path.
Perhaps you should add the directory containing \`secret_prov_gramine.pc'
to the PKG_CONFIG_PATH environment variable
No package 'secret_prov_gramine' found
Package secret_prov_gramine was not found in the pkg-config search path.
Perhaps you should add the directory containing `secret_prov_gramine.pc'
to the PKG_CONFIG_PATH environment variable
No package 'secret_prov_gramine' found
cc secret_prov/client.c -O2 -fPIE -Wall -std=c11 -pie -Wl,--enable-new-dtags -lsecret_prov_attest -o secret_prov/client
secret_prov/client.c:12:10: fatal error: secret_prov.h: File or directory not found
12 | #include "secret_prov.h"
| ^~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:64: secret_prov/client] Error 1

Can you help solve this problem? secret_prov_gramine.pc is nowhere to be found... It does not appear using the locate command... Therefore, I was not able to add its directory to the environment variable.

**Comment by dimakuv** [comment, marked as answer]
Sorry, the PyTorch tutorial was updated **after** the latest stable version (v1.5) was released.

Please follow this link: https://gramine.readthedocs.io/en/latest/tutorials/pytorch/index.html
